AI/DavinciProxy.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace FFXIVVenues.Veni.AI
+{
+    internal class DavinciProxy
+    {
+        static double maxTokens { get; } = 150;
+        double temperature { get; set; } = 0.7;
```

Comment:
Make sure you explicitly say if these two members are `public` or `private`. It's not immediately clear the scope of them, and that impacts on the naming convention too.

Reply:
oh? 
yes those are only used inside this scope so I think it should be private